src/Sql/Sql/help/Get-AzSqlDatabaseAuditing.md

```diff
@@ -9,6 +9,7 @@ schema: 2.0.0
 # Get-AzSqlDatabaseAuditing
 
 ## SYNOPSIS
+### Important: This cmdlet is deprecated, [Get-AzSqlDatbaseAudit](https://docs.microsoft.com/en-us/powershell/module/az.sql/get-azsqldatabaseaudit) is replacing it.
```

Comment:
@bashahee adding additional headers like this unfortunately breaks the help generation script we're running to convert the markdown files into the expected `dll-help.xml` files (as seen in the build failures). Would you mind removing the `###` at the beginning of this line? The rest of the content you added can remain.